README.md

```diff
@@ -73,6 +73,24 @@ Daarnaast zijn voor de software, de ontwikkeling en het gebruik onder andere de
 
 Lees voor meer informatie het bestand `README.md` in elke directory.
 
+## Starten met Abacus
+
+De functionaliteit is nog beperkt, maar het is al mogelijk om Abacus zelf te proberen en te testen. De basisflow invoeren-optellen-PDF werkt al. In de [documentatie](https://github.com/kiesraad/abacus/blob/main/documentatie/functionaliteit/functionaliteit-eisen-en-wensen.md) en op het [board](https://github.com/orgs/kiesraad/projects/1) zie je waar we mee bezig zijn. 
+
+Er zijn een aantal opties beschikbaar om te zien waar we staan: 
+
+### Frontend
```

Comment:
Is het handig / slim om hier de kanttekening bij te maken dat de uiteindelijke Abacus niet via internet werkt?